tools/proofers/review_work.php

```diff
@@ -82,11 +82,11 @@
 echo  "</tr>";
 echo  "<tr>";
 echo   "<td>" . _("Max days since last save") . "</td>";
-echo   "<td><input name='days' type='text' size='4' value='$days'></td>";
+echo   "<td><input name='days' type='number' min='0' value='$days'></td>";
```

Comment:
Is there any reason someone might put 0 as `days`? Maybe `min='1'` would be more appropriate for `days`? 

Should we also add a maximum value constraint?

Reply:
I'm not sure if there's a great use-case for zero as a valid option, but it works (and with my luck someone uses it that way :)

Ditto maximum value - both could be very large and still work so I'm not sure there's a good reason to limit them.